qm2/scf.py

```diff
@@ -0,0 +1,115 @@
+import numpy as np
+import psi4
+
+np.set_printoptions(suppress=True, precision=4)
+
+
+def buildgeom:
+    mol = psi4.geometry("""
+    O
+    H 1 1.1
+    H 1 1.1 2 104
+    """)
+
+    # Build a molecule
+    mol.update_geometry()
+    mol.print_out()
+
+
+    e_conv = 1.e-6
```

Comment:
Do you think this code block should live within the scope of this function only? Or should it live in the scope of the program as a whole? Since we use e_conv, d_conv, damp_value, etc. at later points in the program, we probably want them to live outside of the scope of this function only. In other words, the build_geom function should focus on building the geometry only and not specifying all the conversion parameters, damping parameters, etc.